Guidelines.md

```diff
@@ -1623,8 +1670,82 @@ A recommended minimum retention time is 24 hours.
 
 Operations SHOULD transition to "tombstone" for an additional period of time prior to being purged from the system.
 
-## 14 Push notifications via webhooks
-### 14.1 Scope
+## 14. Throttling, Quotas, and Limits
+### 14.1. Principles
+Services should be as responsive as possible, so as not to block callers. As a rule of thumb any API call that is expected to take longer than 0.5 seconds in the 99th percentile, should consider using the Long-running Operations pattern for those calls. Obviously, services cannot guarantee these response times in the face of potentially unlimited load from callers. Services should therefore design and document call request limits for clients, and respond with appropriate, actionable errors and error messages if these limits are exceeded. Services should respond quickly with an error when they are generally overloaded, rather than simply respond slowly. Finally, many services will have quotas on calls, perhaps a number of operations per hour or day, usually related to a service plan or price. When these quotas are exceeded services must also provide immediate, actionable errors. Quotas and Limits should be scoped to a customer unit: a subscription, a tenant, an application, a plan, or without any other identification a range of ip addresses…as appropriate to the service goals so that the load is properly shared and one unit is not interfering with another.
```

Comment:
this document generally follows the one-sentence-per-line convention, which makes diffs easier to parse in the future. can you please put sentences on their own lines?